apps/frontend/src/components/modals/SpookyModal.tsx

```diff
@@ -34,75 +34,77 @@ export interface ModalProps
   closable?: boolean;
   onClose?: () => void;
   withBackButton?: boolean;
+  username: string;
 }
 
 type ActivityKey =
-  | "trickOrTreat"
-  | "danceOff"
-  | "hideAndSeek"
-  | "nameSwap"
   | "deepConversation"
+  | "danceOff"
+  | "talkWork"
   | "congoLine"
-  | "pairFortune"
   | "beNPCs"
-  | "meetNewPeople";
+  | "pairFortune"
+  | "teamUp"
+  | "chillAndVibe"
+  | "introverse";
 
 const activityMappings: Record<ActivityKey, any> = {
-  trickOrTreat: {
-    emoji: "🔍",
-    label: "trick or treat",
+  deepConversation: {
+    emoji: "💭",
+    label: "have deep conversation",
   },
   danceOff: {
     emoji: "🕺",
     label: "dance off",
   },
-  hideAndSeek: {
-    emoji: "🙈",
-    label: "hide & seek",
-  },
-  nameSwap: {
-    emoji: "👑",
-    label: "name swap",
-  },
-  deepConversation: {
-    emoji: "💭",
-    label: "deep conversation",
+  talkWork: {
+    emoji: "💼",
+    label: "talk about work"
   },
   congoLine: {
     emoji: "👯",
-    label: "congo line",
-  },
-  pairFortune: {
-    emoji: "🔮",
-    label: "pair fortune",
+    label: "form a congo line",
   },
   beNPCs: {
     emoji: "👤",
     label: "be NPCs",
   },
-  meetNewPeople: {
-    emoji: "🤝",
-    label: "meet new people as a team",
+  pairFortune: {
+    emoji: "🔮",
+    label: "get a joint fortune",
+  },
+  teamUp: {
+    emoji: "👼",
+    label: "stick together at the party"
+  },
+  chillAndVibe: {
+    emoji: "😎",
+    label: "chill and vibe"
   },
+  introverse: {
+    emoji: "🎴",
+    label: "play the Introverse card game"
+  }
```

Comment:
<img width="456" alt="image" src="https://github.com/user-attachments/assets/a32e22af-ee98-4ce6-b560-1959a46eeb0f">

Updated to be in line with the current design in freeze V1.2.